pkg/sysregistriesv2/system_registries_v2.go

```diff
@@ -149,17 +149,30 @@ func (config *V1RegistriesConf) Nonempty() bool {
 		len(config.V1TOMLConfig.Block.Registries) != 0)
 }
 
+// aliasesConf includes a map of aliases.
+type aliasesConf struct {
+	// A map of aliases with keys being the alias names, replaced by the
+	// value.
+	Aliases map[string]string `toml:"aliases"`
+}
+
 // V2RegistriesConf is the sysregistries v2 configuration format.
 type V2RegistriesConf struct {
 	Registries []Registry `toml:"registry"`
 	// An array of host[:port] (not prefix!) entries to use for resolving unqualified image references
 	UnqualifiedSearchRegistries []string `toml:"unqualified-search-registries"`
+	// Aliasing sets the aliasing mode (enabled, disabled, etc.).
+	Aliasing string `toml:"aliasing"`
```

Comment:
- I’m not too happy about the “aliasing” name. If this controls non-alias behavior of Podman etc. as well, maybe `short-name-{behavior, handling}`?
- The allowed values, their semantics, and validation (rejecting typos and other values) will need to be nailed down before this is released.

Reply:
Yes let's go with ShortName.  In case there is a future verion alias or people get confused between mirrors and alias.